03_gamedesign/index.tex

```diff
@@ -1,5 +1,79 @@
+
+\end{definition}
+
+\begin{definition} \label{def:gameaction}
+    An \textbf{action} is a decision an island can make or that is made by any Inter-Island Organisation that updates the game state. Actions can be made at different stages within a turn both within Inter-Island Organisations and at the end of a turn. This includes but is not limited to taking resources from or donating resources to from the common pool, rule changes in the IIGO, and gift requests or acceptances. 
+\end{definition}
+
+\begin{definition} \label{def:critical}
+    A \textbf{critical} island is defined as an island whose resources are below the minimum threshold. When this occurs, an island is allowed a grace period during which it is expected to request gifts from other islands in order to reach the minimum amount of resources required to stay in the game.
+\end{definition}
+
+\begin{definition} \label{def:death}
+    A \textbf{death} occurs when an island was in a \textbf{critical} state for $N$ turns. The exact number of turns affects the difficulty of the game so it was investigated and is discussed in simulations and results section. Once an island is dead, it can no longer participate in the simulation.
+\end{definition}
+
+\begin{definition} \label{def:gameover}
+    \textbf{Game over} is defined as the end of the simulation and occurs when all islands have died or the simulation is completed - whichever occurs first.
+\end{definition}
+
+Figure~\ref{fig:gamedesign-flow} depicts a high level diagram of the general flow and structure of the game.
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics{03_gamedesign/images/gamespec-flow.png}
+    \caption{High Level Game Specification Flow Diagram}
+    \label{fig:gamedesign-flow}
+\end{figure}
+
+\subsection{Design Choice Justification}
+Every action taken in the game adds to the dilemma placed on the agent to explore how they interact and attempt to overcome challenges through self-organisation. To complicate matters further, the meetings occur only to form agreements, which can also be broken after they are made. The allocation also occurs first to assist islands that are struggling by allowing them to take from the common pool immediately and use these resources to forage or, at a minimum, pay their tax.
+
+
+\subsection{Game Parameters}
+Many of the parameters within the game can be modified to adjust the difficulty of the game. It should be noted that some of these parameters have a greater impact than others. For example, the agents may or may not have access to the common pool threshold level and this knowledge, or lack thereof, will result in entirely different agent strategies, by forcing agents to rely more on social strategies and cooperation to mitigate unknown risk.

```

Comment:
So slight nitpick here but, I don't think its gonna be so much that the agents will have different strategies its more that agents with certain strategies will be more likely to succeed under certain conditions. I.e. if deer is weaker in one simulation, team 1 will only adjust its strategy slightly (since we mainly focus on deer) but a team who focuses more on fishing will come out ahead overall, anyways. I'd say making that distinction makes it seem like we understand the limits of our agents more, and sets a better expectation

Reply:
really good point actually I've added the distinction thanks :)